app/templates/overview/contributing.hbs

```diff
@@ -0,0 +1,152 @@
+<h1>Contributing</h1>
+
+<section>
+	<h2>Who can contribute</h2>
+	<p>Everyone can contribute. No specific team is the owner of this project and everyone is encouraged to improve and expand it.</p>
+</section>
+
+<section>
+	<h2>How to report issues</h2>
+	<p>
+		To report an issue please create Jira ticket in the MAIN project and select Design System component.
+		You’re free and encouraged to fix the bug yourself.
+	</p>
+</section>
+
+<section>
+	<h2>How to develop</h2>
+	<p>If you want to contribute to DS follow the steps:</p>
+	<ul>
+		<li>Clone <a href="https://github.com/Wikia/design-system">Design System</a> repository</li>
+		<li>Install dependencies with <code class="format-code">npm run setup</code></li>
+		<li>Run <code class="format-code">npm run dev</code> to build the Reference Page and start file watchers</li>
+		<li>Open <a href="http://localhost:4200">localhost</a> to see the Reference Page</li>
+		<li>Modify the Design System or Reference Page source files as you see fit, Ember CLI will rebuild the RP app and reload the page</li>
+		<li>If you want to present the results of your work using devbox see "How to deploy Reference Page to devbox" section below</li>
```

Comment:
Turns out Ember doesn't support anchor links when using `locationType: 'hash'`. We would need to use something like https://github.com/mike-north/ember-anchor and I don't think now is a good moment for that.